src/notation/internal/notationinteraction.cpp

```diff
@@ -3025,11 +3025,7 @@ void NotationInteraction::changeTextCursorPosition(const PointF& newCursorPos)
 
     textEl->mousePress(m_editData);
     if (m_editData.buttons == mu::engraving::MiddleButton) {
-        #if defined(Q_OS_MAC) || defined(Q_OS_WIN)
         QClipboard::Mode mode = QClipboard::Clipboard;
-        #else
-        QClipboard::Mode mode = QClipboard::Selection;
-        #endif
         QString txt = QGuiApplication::clipboard()->text(mode);
```

Comment:
We don't need the `mode` variable anymore at all: `QClipboard::Clipboard` is already the default mode for `QClipboard::text()`. So we can simply call `QGuiApplication::clipboard()->text()`.

Reply:
Will so, tomorrow.

Reply:
I love fixing bug by removing code ;-)